### DISCUSSION TITLE
How to disable Bracketed Paste Mode?

### DISCUSSION THREAD

**Discussion by mzz2017** [linuxdeepin/developer-center#2972] [Features Request & Ideas | 特性请求 & 头脑风暴]
Sometimes I encountered weird "password error".

After my investigation, I found that ~0 and ~1 wrapped the text I pasted, such as `~0mypassword~1`. And I found that this function is called `Bracketed Paste Mode`.

So how to disable "Bracketed Paste Mode" in deepin-terminal?

**Comment by ch593030323** [comment]
U can modify the source code directly in "3rdparty/terminalwidget/lib/TerminalDisplay.cpp".

**Comment by mzz2017** [comment]
Thanks. But it would be better if there is any configuration option.

**Comment by ch593030323** [comment]
Sorry, there is not configuration option or settings for this.
We will consider this function reasonably.

**Comment by mzz2017** [comment]
Thanks. It is the best terminal I have used.

**Comment by Sky-Asher** [comment]
Thank you for your support, we will do better and better.
